@tus/azure-store is missing dist folder in distributed package leading to import issues

### Initial checklist

- [X] I understand this is a bug report and questions should be posted in the [Community Forum](https://community.transloadit.com/c/tus/6)
- [X] I searched [issues](https://github.com/tus/tus-node-server/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### Steps to reproduce

Unlike all the other subpackages, `@tus/azure-store` doesn't contain the dist dir (see https://www.npmjs.com/package/@tus/azure-store?activeTab=code for example), which leads to import issues, at least in Typescript projects, but probably also in vanilla JS. I've prepared a minimal reproducible example here: https://github.com/fdebijl/tus-repro

### Expected behavior

Importing from `@tus/azure-store` should be possible without any errors. E.g.: `import { AzureStore } from '@tus/azure-store';`

### Actual behavior

The Typescript compiler errors out with `Cannot find module '@tus/azure-store' or its corresponding type declarations.` when adding an import from `@tus/azure-store`. Importing from `@tus/azure-store/src` is possible, but obviously a bad practices.

--- a/.changeset/friendly-panthers-know.md
+++ b/.changeset/friendly-panthers-know.md
@@ -1,0 +1,5 @@
+---
+"@tus/azure-store": patch
+---
+
+Correctly publish dist folder

--- a/test/tsconfig.json
+++ b/test/tsconfig.json
@@ -1,6 +1,7 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig.json",
   "references": [
+    { "path": "../packages/azure-store/tsconfig.build.json" },
     { "path": "../packages/file-store/tsconfig.build.json" },
     { "path": "../packages/gcs-store/tsconfig.build.json" },
     { "path": "../packages/s3-store/tsconfig.build.json" },

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -2,6 +2,7 @@
   "$schema": "https://json.schemastore.org/tsconfig.json",
   "files": [],
   "references": [
+    { "path": "packages/azure-store/tsconfig.json" },
     { "path": "packages/file-store/tsconfig.json" },
     { "path": "packages/gcs-store/tsconfig.json" },
     { "path": "packages/s3-store/tsconfig.json" },

